lst_scripts/onsite_mc_merge_and_copy_dl1.py

```diff
@@ -148,7 +148,11 @@ def main(input_dir, flag_full_workflow=False, particle2jobs_dict={}, particle=No
         for set_type in ['testing', 'training']:
             tdir = os.path.join(running_DL1_dir, set_type)
             output_filename = 'dl1_'
-            for i in [-4, -3, -2, -1]:
+            if '/zenith_' in running_DL1_dir:  # /20200629_prod5/{particle}/zenith_20deg/south_pointing/{prod_id}/DL1
+                list_find_better_way_to_do_this = [-5, -4, -3, -2, -1]
+            else:                              # /20190415/{particle}/south_pointing/{prod_id}/DL1
+                list_find_better_way_to_do_this = [-4, -3, -2, -1]
+            for i in list_find_better_way_to_do_this:
                 output_filename += running_DL1_dir.split('/')[i]
                 output_filename += '_'
```

Comment:
what about using the name of the simtel files?
```
anyfile = os.listdir(tdir)[0]
output_filename = re.sub(r'.simtel.*', '.h5', re.sub(r'run\w_', 'merged', anyfile))
```
`re.sub(r'run\w_', 'merged', anyfile)` replaces `run*_` with `merged`
`re.sub(r'.simtel.*', '.h5'` replaces simtel extension with `h5`

Reply:
No `.simtel.*` in sub-dl1 files (they are all already .h5 files). Thanks for the comment in any case, I will change they way to create the output file.

Reply:
If we add `merged` to the file name, it will be kept when converted to dl2.

Reply:
Ha yes, mi stupido.

Reply:
right, good thinking, so let's just remove this run bad boy and keep all the rest intact
`re.sub(r'run\w__', '', anyfile)`

Reply:
personally I will go for an output name of the kind;
`dl1_proton_20deg_180deg_{prod_id}_{test/training}.h5` 

However, for the cas of the gammas, how would you prefer ? 
 a) `/off0.Xdeg/dl1_gamma_20deg_180deg_{prod_id}_{test/training}.h5` 
 b  `dl1_gamma_20deg_180deg_{off0.Xdeg}_{prod_id}_{test/training}.h5`

Reply:
so keeping the previous naming trick? very error-prone and non-flexible imo.

for gammas the off should definitely be in the filename (and it would, from the simtel filename ;-) )

Reply:
gammas sub-dl1 output names;
`dl1_gamma_20deg_180deg_run9___cta-prod5-lapalma_4LSTs_MAGIC_desert-2158m_4LSTs_MAGIC_mono.h5`
and simtel;
`gamma_20deg_180deg_run9___cta-prod5-lapalma_4LSTs_MAGIC_desert-2158m_4LSTs_MAGIC_mono.simtel.gz`
🤔

Reply:
Sorry, the thread was not uploading (just saw the messages by the email notifications). If you prefer I can keep the `re.sub()` method :)

Reply:
ok, it's only in the `off0.4`:
`gamma_20deg_180deg_run1___cta-prod5-lapalma_4LSTs_MAGIC_desert-2158m_4LSTs_MAGIC_mono_off0.4.simtel.gz`
🙄

Reply:
Ah, nice seen @vuillaut, thanks !